packages/documentation/src/stories/utilities/spacing/spacing.stories.ts

```diff
@@ -1,15 +1,165 @@
+      control: {
+        type: 'select',
+        labels: positionOptions,
+      },
+      options: Object.keys(positionOptions),
+      table: {
+        category: 'General',
+      },
+    },
+    paddingSize: {
+      name: 'Padding size',
+      description: 'Sets the size of the Padding.',
+      control: {
+        type: 'select',
+      },
+      options: sizingOptions,
+      table: {
+        category: 'General',
+      },
+    },
+    paddingPosition: {
+      name: 'Pading Position',
+      description: 'Sets the position of the Padding.',
+      control: {
+        type: 'select',
+        labels: positionOptions,
+      },
+      options: Object.keys(positionOptions),
+      table: {
+        category: 'General',
+      },
+    },
   },
 };
 
 export default meta;
 
 type Story = StoryObj;
 
-export const Default: Story = {};
+export const Default: Story = {
+  render: (args: Args) => {
+    return html`
+      <!-- Demo -->
+      <div class="bg-primary p-regular d-flex align-items-start">
+        <div class="flex-fill d-flex">
+          <div class="bg-petrol">
+            <div
+              class="bg-petrol-bright border border-dark h-bigger-giant m${args.marginPosition}-${args.marginSize} p${args.paddingPosition}-${args.paddingSize} w-bigger-giant"
+            >
+              <div class="bg-light h-100"></div>
+            </div>
+          </div>
+        </div>
+
+        <!-- Legend -->
+        <ul class="ps-regular text-white">
+          <li class="d-flex align-items-center mb-regular">
+            <div class="bg-petrol h-regular w-regular me-mini"></div>
+            <span>margin</span>
+          </li>
+          <li class="d-flex align-items-center mb-regular">
+            <div class="bg-petrol-bright h-regular w-regular me-mini"></div>
+            <span>padding</span>
+          </li>
+          <li class="d-flex align-items-center mb-regular">
+            <div class="bg-light h-regular w-regular me-mini"></div>
+            <span>content</span>
+          </li>
+        </ul>
+      </div>
+    `;
+  },
+};
+
+export const responsiveExample: Story = {
+  render: (args: Args) => {
+    return html`
+      <div class="bg-primary p-regular">
+        <div
+          class="border border-dark bg-petrol-bright h-bigger-giant w-bigger-giant p-regular p-lg-big"
+        >
+          <div class="bg-light h-100"></div>
+        </div>
+        <p class="text-white"><small>Resize the browser window to see changes.</small></p>
+      </div>
+    `;
+  },
+};
+
+export const automaticResponsiveExample: Story = {
+  render: (args: Args) => {
+    return html`
+      <div class="bg-primary p-regular">
+        <div class="border border-dark bg-petrol-bright h-bigger-giant w-bigger-giant p-large-r">
+          <div class="bg-light h-100"></div>
+        </div>
+        <p class="text-white"><small>Resize the browser window to see changes.</small></p>
+      </div>
+    `;
+  },
+};
```

Comment:
Try to adopt the above example here also.
Put everything in a decorator, that should not be visible in the code preview.